ts/features/idpay/initiative/configuration/xstate/machine.ts

```diff
@@ -375,7 +408,14 @@ const createIDPayInitiativeConfigurationMachine = () =>
         })),
         confirmIbanOnboarding: assign((_, event) => ({
           ibanBody: event.ibanBody
-        }))
+        })),
+        showFailureToast: _ =>
+          _.errorState !== undefined
+            ? showToast(
+                I18n.t(`idpay.configuration.failureStates.${_.errorState}`),
+                "danger"
+              )
+            : null
```

Comment:
Maybe we can move this action in the `actions.ts` file, it would be more testable and eave "embedded" actions for context mutations. What do you think?